spec/system/page_objects/components/composer.rb

```diff
@@ -54,6 +59,26 @@ def button_label
         find("#{COMPOSER_ID} .btn-primary .d-button-label")
       end
 
+      def emoji_picker
+        find("#{COMPOSER_ID} .emoji-picker")
+      end
+
+      def emoji_autocomplete
+        find(".autocomplete.ac-emoji")
+      end
+
+      def has_emoji_autocomplete?
+        has_css?(".autocomplete.ac-emoji")
+      end
+
+      def has_emoji_suggestion?(emoji)
+        has_css?(".autocomplete.ac-emoji .emoji-shortname", text: emoji)
+      end
```

Comment:
Minor but we're repeating `".autocomplete.ac-emoji"` alot here. Perhaps we should extract the duplication into a constant to DRY the code a little here?